.github/workflows/linting.yml

```diff
@@ -3,7 +3,7 @@ name: Linting
 on: [push, pull_request]
 
 jobs:
-  flake8:
+  pre-commit:
```

Comment:
this file is the only change of this PR, all the rest it automatically generated corrections :otter: